### DISCUSSION TITLE
ComboBox DoubleTapped event

### DISCUSSION THREAD

**Discussion by Agarvaen** [AvaloniaUI/Avalonia#13223] [Q&A] [answered]
Hello everyone! I want to open a window on combobox double click. So I find a DoubleTapped event and think: this is it. But no (
If I use touchpad then event firing as expected, but if use double left click then event is not firring.
When I use a mouse only one PointerPressed event is generated and so no DoubleTapped event...
Seems like a bug.
OS: Windows 10
Avalonia: 11.0.4

Any help/suggestions?

**Comment by timunie** [comment, marked as answer]
PointerPressed has e.ClickCount irrc. but imo, this control is not designed to work like that. Personally, I'd use a SplitButton here. 

https://docs.avaloniaui.net/docs/next/reference/controls/buttons/splitbutton

**Comment by Agarvaen** [comment]
Yes e.ClickCount may help. I forgot about it. Thank you!
I don't think I can use SpliteButton in my case. Combobox is inside a DataGrid cell and have bound ItemsSource and SelectedItem.

**Reply by timunie** [reply]
`DataGridTemplateColumn`. In SplitButton Flyout, you can add a ListBox with ItemsSource and SelectedItem. Content also bind to Selected Item

**Reply by Agarvaen** [reply]
Hm... Interesting. Need to try it.  Thanks again!
